src/render.js

```diff
@@ -133,12 +134,13 @@ const render = async function(
     // Here we get the existing VM context for this request or create a new one
     // and configure it accordingly.
     const context = getOrCreateRenderContext(
+        globals ? globals["location"] : "http://www.khanacademy.org",
```

Comment:
Maybe webapp should be passing this fallback location as part of the request.

Reply:
Hmm, I don't think so. If anything, we should move `location` off of `globals` and make it a mandatory field in the request. This is just a workaround for now to ensure the default behaviour hasn't changed.

Reply:
Does this make it hard for people to use react-render-server for things other than `khanacademy.org`?

Reply:
No harder than it was before. This diff doesn't change anything in this area compared with before this change.

Reply:
I missed that we had `url:  "http://www.khanacademy.org",` set before.  I thought this fallback was something new, but it appears that it isn't.